scripts/db_migrations/v2.8/addIssueIdToGroups.js

```diff
@@ -0,0 +1,36 @@
+// mongo --host hostname --port port -u username -p password --authenticationDatabase db scripts/db_migrations/v2.8/addIssueIdToGroups.js
+
+print('adding issue IDs to groups');
+
+var addIssueIdToGroup = function(myDb, settingId, groupId, issueId) {
+	print('adding ' + issueId + ' to ' + groupId);
+	myDb.getCollection(settingId + '.groups').update({ _id: groupId }, { $set: { issue_id: issueId } });
+};
+
+db.getSiblingDB('admin').adminCommand({listDatabases:1}).databases.forEach(function(database) {
+	var myDb = db.getSiblingDB(database.name);
+	print('DB: ' + database.name + ' ----------------------------------');
+	myDb.getCollection('settings').find().forEach(function(setting) {
+		print('Model: ' + setting._id);
+		myDb.getCollection(setting._id + '.issues').find().forEach(function(issue) {
+			print('Issue: ' + issue.name);
```

Comment:
too many prints! will take absolutely forever to run. remove this one (printing db name and model will do)